src/theory/sep/theory_sep.cpp

```diff
@@ -1338,13 +1303,99 @@ Node TheorySep::getLabel( Node atom, int child, Node lbl ) {
     //TypeNode ltn = NodeManager::currentNM()->mkSetType(NodeManager::currentNM()->mkRefType(refType));
     Node n_lbl = sm->mkDummySkolem(ss.str(), ltn, "sep label");
     d_label_map[atom][lbl][child] = n_lbl;
-    d_label_map_parent[n_lbl] = lbl;
```

Comment:
It seems this map was never read. What was the point of having it in the first place?

Reply:
It wasn't used before, although it will be used in my next PR.